### DISCUSSION TITLE
Does all operations are lossless?

### DISCUSSION THREAD

**Discussion by necros2k7** [zeux/meshoptimizer#408] [Q&A] [answered]
ie. gltfpack -i 1.gltf -o out.gltf
-tc -tu -tq -mm options?

**Comment by zeux** [comment, marked as answer]
By default gltfpack uses quantization on all geometry components which is a lossy process. You can disable it with `-noq`, which will leave geometry operations that are lossless (although this is not recommended because it increases file size / memory impact).

By default gltfpack doesn't compress textures, but when `-tc` or other texture options are used, it uses Basis Universal format to compress textures which is also lossy.

`-mm` and `-mi` are lossless as they just transform draw data to reduce draw call count.
